content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,670 @@
+  options such as the K8s cluster receiver and Kubelet stats receiver.
+
+We'll also do a technical dive into the OTel Operator's Target Allocator (TA)
+and learn:
+
+- How it can be used for Prometheus service discovery.
+- How it ensures even distribution of Prometheus targets.
+
+## OTel and Prometheus
+
+Since OTel is primarily focused on the instrumentation part of observability, it
+doesn't provide a backend for storing telemetry; you have to forward the data to
+a backend vendor for storage, alerting, and querying.
+
+Prometheus, on the other hand, provides a time-series data store you can use for
+your metrics, in addition to instrumentation clients. You can view graphs and
+charts, set up alerts, and query your data via their web user interface. It also
+encompasses a data format, known as
+[Prometheus text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats).
+
+Prometheus _data_ is stored as a dimensional time-series, meaning that the data
+has attributes (for example, labels or dimensions) and a timestamp.
+
+The Prometheus _server_ collects Prometheus metrics data from targets defined in
+a configuration file. A target is an endpoint that exposes metrics for the
+Prometheus server to store.
+
+Prometheus is so ubiquitous in the monitoring space that many tools natively
+emit metrics in Prometheus format, including
+[Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/)
+and
+[HashiCorp's Nomad](https://developer.hashicorp.com/nomad/docs/operations/monitoring-nomad).
+And for those that don’t, there are a number of vendor- and community-built
+[Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/) to
+aggregate and import data into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and
+application metrics, one of its most popular use cases is to monitor Kubernetes.
+This is the aspect of Prometheus monitoring that we will focus on in this
+article.
+
+## Prometheus metrics with OpenTelemetry
+
+In this section, you'll learn about a couple of OTel Collector components that
+demonstrate the interoperability between OTel and Prometheus.
+
+First, let’s do a quick refresher on the [Collector](/docs/collector/) – it’s an
+OTel component that can be used to collect telemetry from multiple sources and
+export data to multiple destinations. The Collector also handles telemetry
+processing, such as modifying data attributes and scrubbing personally
+identifiable information. For example, you can use Prometheus SDKs to generate
+metrics, ingest them with the Collector, do some processing (if desired) and
+then forward them to your chosen backend.
+
+![Diagram showing the different components of the OTel Collector](OTel-collector-refresher.png)
+
+The
+[Prometheus receiver](/docs/kubernetes/collector/components/#prometheus-receiver)
```

Comment:
```suggestion
[Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)
```

That page does not exist and breaks the URL check

Reply:
That's odd, it does exist... https://opentelemetry.io/docs/kubernetes/collector/components/#prometheus-receiver  but I replaced it with your suggestion.

Reply:
hm, maybe I did a mistake here and the problem was with the host metrics receiver from the start, but let's leave it like that for now, I just want to get this PR working :-)